packages/salesforcedx-utils-vscode/src/telemetry/index.ts

```diff
@@ -4,6 +4,5 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { TelemetryService } from './telemetry';
-export const telemetryService = TelemetryService.getInstance();
-export { TelemetryBuilder, TelemetryData, Properties, Measurements } from './telemetry';
+
+export { TelemetryService, TelemetryBuilder, TelemetryData, Properties, Measurements } from './telemetry';
```

Comment:
I had to remove `TelemetryService.getInstance()` from here cause it was causing webpack to produce some duplicate imports. We'll take a second look at this in the next days but for now I've made the necessary updates for this to work.